README.md

```diff
@@ -32,16 +32,24 @@ set of technologies that enable big data systems to process and move data fast.
 
 Major components of the project include:
 
- - [The Arrow Columnar In-Memory Format](https://github.com/apache/arrow/tree/master/format)
+ - [The Arrow Columnar In-Memory Format](https://github.com/apache/arrow/tree/master/format):
+   a standard and efficient in-memory representation of various datatypes, plain or nested
+ - [The Arrow IPC Format](https://github.com/apache/arrow/tree/master/format):
```

Comment:
Can/should this link to something more specific? It's the same link as the previous line.

Reply:
Well, what should it link to? There isn't a single file for the IPC format.

Reply:
http://arrow.apache.org/docs/format/Columnar.html#serialization-and-interprocess-communication-ipc or the equivalent location in the source tree?

Reply:
Since the format is stable, it should be safe to link to the published docs here, right?